src/multibody/joint/joint-free-flyer.hpp

```diff
@@ -275,13 +278,42 @@ namespace se3
       return result; 
     }
 
-    ConfigVector_t random_impl(const ConfigVector_t & , const ConfigVector_t & ) const
+    ConfigVector_t random_impl() const
     { 
       ConfigVector_t q(ConfigVector_t::Random());
       q.segment<4>(3).normalize();// /= q.segment<4>(3).norm();
       return q;
     } 
 
+    ConfigVector_t uniformlySample_impl(const ConfigVector_t & lower_pos_limit, const ConfigVector_t & upper_pos_limit ) const
```

Comment:
In the function signature or in the doc ?
http://www.gotw.ca/publications/mill22.htm